server/src/test/java/org/cloudfoundry/identity/uaa/authentication/BackwardsCompatibleTokenEndpointAuthenticationFilterTest.java

```diff
@@ -49,12 +49,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.same;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyZeroInteractions;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
please avoid start import
https://jiga.dev/avoiding-wildcard-imports-in-java-kotlin-with-intellij/

Reply:
Can I add additional commits? Will my commits be squashed?

Reply:
yes finally PRs will be squashed to develop branch , so for your work you can provide single commits